tests/Unit/Api/V1/HasSubscriptionTraitTest.php

```
@@ -134,15 +134,17 @@ private function makeSubscription(
         ?int $paddlePlan = null,
         mixed $nextPayment = null,
+        ?string $paddleStatus = null,
     ): object {
-        return new class($valid, $recurring, $onGracePeriod, $paddlePlan, $nextPayment)
+        return new class($valid, $recurring, $onGracePeriod, $paddlePlan, $nextPayment, $paddleStatus)
         {
             public function __construct(
                 private readonly ?bool $valid,
                 private readonly ?bool $recurring,
                 private readonly ?bool $onGracePeriod,
                 public ?int $paddle_plan,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$paddle_plan'. (undefined)

(UnusedFormalParameter)